packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -423,6 +423,16 @@ class CloudFormation {
     });
   }
 
+  async listStackResources(stackId) {
+    const meta = stateManager.getMeta();
+    stackId = stackId || _.get(meta, ['providers', providerName, 'StackName'], undefined);
+    if (!stackId) {
+      throw new Error(`StackId not found in amplify-meta for provider ${providerName}`);
+    }
+    // StackName param can be a StackName, StackId, or a PhysicalResourceId
+    return this.cfn.listStackResources({ StackName: stackId }).promise();
```

Comment:
Yes.

Reply:
Updated IAM docs as part of docs PR: https://github.com/aws-amplify/docs/pull/3302

`cloudforamtion:ListStackResources` is the only new permission. It will need to be updated in the services mentioned.